monai/handlers/mlflow_handler.py

```diff
@@ -88,6 +100,11 @@ def __init__(
         global_epoch_transform: Callable = lambda x: x,
         state_attributes: Optional[Sequence[str]] = None,
```

Comment:
I didn't see you change the default value of this arg to `["max_epochs", "epoch_length"]` as my previous comment. Do you agree with that?

Reply:
Hi @Nic-Ma ,
     In my opinion, the `default_attr_name` should be some parameters that would not change through the whole experiment, therefore they should be recorded at the beginning of the training as parameters, as what i did in this code.  And the attributes specified in `state_attributes` are kind of epoch depending things that would expect to change every epoch. Thus, these parameters shoud be recorded every epoch as metrics. I would suggest we keep these two things split. What would you like?

Thanks,
Bin

Reply:
the values might be changed during FL  https://github.com/Project-MONAI/MONAI/blob/e2fc703ad9467eb2ffe28e0a5c51d252e22cef64/monai/fl/client/monai_algo.py#L495-L496 so I agree with @Nic-Ma

Reply:
Hi @wyli ,

The values might be changed during FL but this PR records the value at "START" event of every FL round, I think it's OK? I think maybe we just need to change the var name to `default_tracking_params` to resolve the confusion?
What do you think?

Thanks.